README.md

```diff
@@ -145,6 +145,7 @@ mysqld.username                            | Username to be used for connecting
 config.my-cnf                              | Path to .my.cnf file to read MySQL credentials from. (default: `~/.my.cnf`)
 log.level                                  | Logging verbosity (default: info)
 exporter.lock_wait_timeout                 | Set a lock_wait_timeout (in seconds) on the connection to avoid long metadata locking. (default: 2)
+exporter.disable_lock_wait_timeout         | Disable the lock_wait_timeout connection parameter. Makes the exporter compatible with older versions of MySQL. (default: false)
```

Comment:
I prefer "positive" flags, so that we don't have double negative behavior.

The disable would be `--no-exporter.enable_lock_wait_timeout`.

```suggestion
exporter.enable_lock_wait_timeout         | Disable the lock_wait_timeout connection parameter. Makes the exporter compatible with older versions of MySQL. (default: true)
```